packages/react-aria-components/stories/ListBox.stories.tsx

```diff
@@ -176,3 +177,14 @@ ListBoxDnd.story = {
     }
   }
 };
+
+
+export const ListBoxHover = () => (
+  <ListBox className={styles.menu} aria-label="test listbox">
+    <MyListBoxItem onHoverStart={action('onHoverStart')} onHoverChange={action('onHoverChange')} onHoverEnd={action('onHoverEnd')}>Hover</MyListBoxItem>
```

Comment:
This item has hover events disabled since it doesn't have an action or selection. If you add actions to these via `onAction={action('onAction')}`, we should be able to see the hover events in the actions panel.

Reply:
I should have addressed that when I fixed the logic.
Thanks for the comment!